.github/workflows/release.yml

```
@@ -12,10 +12,11 @@ jobs:
 
     steps:
       - uses: actions/checkout@v1
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
           node-version: 12.x
           registry-url: 'https://registry.npmjs.org'
+          cache: npm
```

This should be

```suggestion
          cache: yarn
```

as well :)

---

No package installation with `npm` nor `yarn` so I assume cache is not necessary here
```suggestion
```